src/components/DashboardsCard/DashboardsCard.tsx

```diff
@@ -117,10 +130,5 @@ export type DashboardCardOpts = {
 
 export const DashboardsCard = (opts?: DashboardCardOpts) => {
   const { entity } = useEntity();
-
-  return !isGrafanaAvailable(entity) ? (
-    <MissingAnnotationEmptyState annotation={GRAFANA_ANNOTATION_TAG_SELECTOR} />
-  ) : (
-    <Dashboards entity={entity} opts={opts} />
-  );
```

Comment:
I think we should keep this and just require people that want Grafana links to add the basic `grafana/tag-selector` annotation, like we do for our other integrations.

Reply:
Okay, so I will keep it as it was before